client/sidebar/sections/components/OmnichannelCallToggleReady.tsx

```diff
@@ -0,0 +1,71 @@
+import { Sidebar } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement, useEffect, useState } from 'react';
+
+import { useCallClient } from '../../../contexts/CallContext';
+import { useTranslation } from '../../../contexts/TranslationContext';
+
+export const OmnichannelCallToggleReady = (): ReactElement => {
+	const [agentEnabled, setAgentEnabled] = useState(false); // TODO: get from AgentInfo
```

Comment:
not possible yet?

Reply:
Not yet, the idea is to store the state on the agent so the agents don't need to keep activating it every time the page reloads, like we do with the livechat availability